src/index.html

```diff
@@ -1,12 +1,229 @@
 <!doctype html>
-<html lang="en">
+<html lang="ru">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="d">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="head">
+      <a href="#home" class="head__logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link is-hover" data-qa="nav-hover">Ноутбуки и компьютеры</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link">Гаджеты</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link">Планшеты</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link">ФОТО</a>
+          </li>
+          <li>
+            <a href="#" class="nav__link nav_last">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="row1">
+        <div class="card" data-qa="card">
+          <img src="./images/imac.jpeg" alt="d" class="card__logo">
+          <div class="card__title">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </div>
+          <div class="card__code">Код товара: 195434</div>
+          <div class="card__rate">
+            <div class="card__stars star stars--4">
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+              <div class="stars_star"></div>
+            </div>
+            <div class="card__feedback feedback">Отзывов: 5</div>
+          </div>
+          <div class="card__price">
+            <div class="card__text text">Цена:</div>
+            <div class="card__num num">69 999 грн</div>
+          </div>
+          <a href="/" class="card__link link" data-qa="card-hover">Купить</a>
```

Comment:
Now if I click on the button, there will be a 404 error
```suggestion
          <a href="#" class="card__link link" data-qa="card-hover">Купить</a>
```